### PR TITLE
[gapi.client.people] Update Google People API

### LINKED ISSUE #43219: Hi, this method gapi.client.people.get from @types/gapi.client.people is returned undefined. When I log "gapi.client.people" the function "get" is inside "people". Should I use "gapi.client.people.people.get()" instead of "gapi.client.people.get()"?
If you know how to fix the issue, make a pull request instead.

- [ ] I tried using the `@types/xxxx` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [ ] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @....

If you do not mention the authors the issue will be ignored.

### PR DESCRIPTION
**Pull request by namoscato**
Fixes #43219

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Maxim-Mazurok/google-api-typings-generator#my-fork-diff
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

ℹ️ These were regenerated via @Maxim-Mazurok's [google-api-typings-generator](https://github.com/Maxim-Mazurok/google-api-typings-generator):

```
$ npm start -- --service people

> google-api-typings-generator@ start /Users/namoscato/dev/git/google-api-typings-generator
> ts-node src/cli.ts --out ./types "--service" "people"

Output directory: ./types
base directory: ./types
typings directory: ./types

Discovering Google services...
Done
Generating people:v1 definitions... 
```

### CODE REVIEW

File: types/gapi.client.people/tslint.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "await-promise": false
+        "interface-name": [false],
```
Comment (uniqueiniquity) [inline types/gapi.client.people/tslint.json:4]: Please suppress this and the other two rules in the source instead of suppressing them globally. Thanks!

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/gapi.client.people/tslint.json (hunk 1 above)
Comment (Maxim-Mazurok) [reply types/gapi.client.people/tslint.json:4]: Will do, thanks!

**Comment by Maxim-Mazurok** [comment]
@namoscato you can go ahead and commit changes to `tslint.json`, #43544 will be reviewed in a couple of days, I hope.

**Comment by Maxim-Mazurok** [comment]
@namoscato I think they will also ask you to remove `await-promise` rule.
It will fail tests now, but after #43544 they will pass

**Comment by namoscato** [comment]
Well, it's stricter now than it was previously and unblocked by #43544 in its current state, which is probably preferable. Happy to make another change if necessary but am going to leave it for now.

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**